uniffi/src/ffi/handle_map.rs

```diff
@@ -0,0 +1,414 @@
+use ffi_support::{ExternError, Handle, HandleError, HandleMap, IntoFfi};
```

Comment:
nit: `handle_map.rs` seems a little unfortunate since it doesn't actually expose an object called HandleMap, even though such a name is used heavily. `arc_handle_map.rs` seems a bit less surprising, but is long. I don't really care about this, but thought it worth mentioning.

Reply:
Renamed to `handle_maps`, because it defines `ArcHandleMap`, `MutexHandleMap` and `UniffiMethodCall`.